src/components/heroSection/HeroSection.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './heroSection.css';
+import imageProfile from '../../assets/image.png';
+
+interface HeroProps {
+  names: string;
+  date: string;
+  location: string;
+}
+
+export const HeroSection = ({ names, date, location }: HeroProps) => {
+  return (
+    <div className="main-container">
```

Comment:
className='hero-container'

Reply:
use section instead of div 

`<section id='hero' className='hero-container' > </section>`